src/codegen.cpp

```diff
@@ -4723,7 +4719,7 @@ static std::unique_ptr<Module> emit_function(jl_lambda_info_t *lam, jl_llvm_func
                                              0,
                                              true,
                                              nullptr);
-                builder.SetCurrentDebugLocation(DebugLoc::get(0, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
+                builder.SetCurrentDebugLocation(DebugLoc::get(inlfline, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
             }
```

Comment:
Typically, the first statements emitted for a function should have a line number of zero. I'm not sure that ends up mattering much right now for us  (due to the way we do inlining), but normally it helps debuggers avoid stopping in uninitiallized stack frames and profilers to  properly account for the cost of the prologue vs. the function body.